landing/index.html

```diff
@@ -3,24 +3,54 @@
     <head>
         <meta charset="utf-8" />
         <meta name="viewport"
-              content="width=device-width, initial-scale=1"
+            content="width=device-width, initial-scale=1"
         />
         <meta name="description"
-              content="Landing for veraPDF viewer"
+            content="Landing for veraPDF viewer"
         />
         <title>veraPDF viewer</title>
         <link rel="shortcut icon" type="image/x-icon" href="./assets/img/favicon.ico">
         <link rel="stylesheet" media="all" href="./assets/css/style.css">
         <script src="index.js"></script>
     </head>
     <body>
+        <header>
+            <div class="header" id="header">
+                <a href="./">
+                    <img src="assets/img/logo.svg" class="logo" alt="logo" />
+                </a>
+                <a class="btn btn-header" id="btn-header" href="/validate" hidden>Try now</a>
+            </div>
+            <div class="header-content">
+                <h2>PDF Accessibility checker</h2>
+                <p id="powered-by">powered by veraPDF</p>
+                <a class="btn" href="/validate" id="try-now-button">Try now</a>
+            </div>
+        </header>
         <main>
-            <img src="assets/img/veraPDF-logo-400.png" class="logo" alt="logo" />
-            <div class="app-info">
-                <h2>veraPDF for WCAG</h2>
-                <div id="app-version-label" hidden>Latest version: <span id="app-version"></span></div>
-				<a class="btn" href="/validate">Start</a>
+            <div class="main-content">
+                <div class="info">
+                    <h3>What does this application do?</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates nesciunt nam inventore reiciendis repellendus corporis iste optio. Ipsum pariatur, minus, earum tempora qui asperiores exercitationem laborum a natus cupiditate iste.</p>
+                </div>
+                <div class="info">
+                    <h3>How to use this application?</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates nesciunt nam inventore reiciendis repellendus corporis iste optio. Ipsum pariatur, minus, earum tempora qui asperiores exercitationem laborum a natus cupiditate iste.</p>
+                </div>
             </div>
+            <a class="read-more" href="/validate/about">
+                Read more
+                <img src="assets/img/arrow.svg" alt="arrow" />
+            </a>
         </main>
+        <footer>
```

Comment:
Footer is hidden by default even if window has enough space and user should scroll down to see it. Let's keep the footer visible if there is enough space